dissect/target/helpers/fsutil.py

```diff
@@ -967,21 +965,45 @@ def resolve_link(
     return entry
 
 
-def decompress_and_read(path: TargetPath) -> bytes:
-    if path.suffix == ".gz":
-        return gzip.GzipFile(fileobj=path.open()).read()
-    elif path.suffix == ".bz2":
-        bytestream = path.open().read()
-        return bz2.decompress(bytestream)
-    else:
-        return path.open().read()
+# def decompress_and_read(path: TargetPath) -> bytes:
```

Comment:
Forgot to remove the commented code I presume.

Reply:
Removed in 6d028f4b6efa6ebd4d6626580db7181ea75c1fe4.